modules/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -101,7 +101,6 @@ func (cs ClientState) VerifyUpgradeAndUpdateState(
 	newClientState := NewClientState(
 		tmUpgradeClient.ChainId, cs.TrustLevel, cs.TrustingPeriod, tmUpgradeClient.UnbondingPeriod,
 		cs.MaxClockDrift, tmUpgradeClient.LatestHeight, tmUpgradeClient.ProofSpecs, tmUpgradeClient.UpgradePath,
-		cs.AllowUpdateAfterExpiry, cs.AllowUpdateAfterMisbehaviour,
```

Comment:
I guess there's a sideaffect here. These parameters will be set to false, false upon upgrades (even if the previous values were true, true). Shouldn't cause an issue, but is worth being aware of

Reply:
i can add an inline doc for this :)

Reply:
added a note in upgrade docs about this